docs/en/UM_Authentication.md

```diff
@@ -9,30 +9,40 @@ layout: page
 
 To maintain the security of your applications and users, all requests to Adobe I/O APIs must be authenticated and authorized using standards such as OAuth and JSON Web Tokens (JWT). Before you can access the User Management API, you will need to obtain access credentials by creating a new Integration in the Adobe I/O Console or Admin Console.
 
-![Diagram of Authentication workflow](AuthWorkflow.png)
```

Comment:
I think this is still useful to have. Shall we update it to OAuth instead?

Reply:
I reinstated at the bottom of the page the JWT Auth for faster removal when needed. I also created a diagram for OAuth S2S.